sp/src/game/server/hl2/npc_combine.cpp

```diff
@@ -3580,6 +3589,16 @@ int CNPC_Combine::SelectSchedule( void )
 			}
 		}
 
+#ifdef EZ2
+		// If we have an obstruction, just plow through
+		if ( HasCondition( COND_COMBINE_OBSTRUCTED ) )
+		{
+			SetTarget( m_hObstructor );
```

Comment:
Forgive me if I missed something, but how do we know the obstructor isn't friendly?